### DISCUSSION TITLE
Error trying to open version 9 RC below (installed all required)

### DISCUSSION THREAD

**Discussion by SkullRunnerGR** [d2phap/ImageGlass#1682] [Q&A]
Version: igcmd v9.0.6.1008
Release code: moon
OS: Microsoft Windows NT 10.0.22635.0 64-bit

----------------------------------------------------
Error:

Unable to cast COM object of type 'System.__ComObject' to interface type 'Microsoft.Web.WebView2.Core.Raw.ICoreWebView2WebMessageReceivedEventArgs2'. This operation failed because the QueryInterface call on the COM component for the interface with IID '{06FC7AB7-C90C-4297-9389-33CA01CF6D5E}' failed due to the following error: No such interface supported (0x80004002 (E_NOINTERFACE)).
----------------------------------------------------

System.InvalidCastException: Unable to cast COM object of type 'System.__ComObject' to interface type 'Microsoft.Web.WebView2.Core.Raw.ICoreWebView2WebMessageReceivedEventArgs2'. This operation failed because the QueryInterface call on the COM component for the interface with IID '{06FC7AB7-C90C-4297-9389-33CA01CF6D5E}' failed due to the following error: No such interface supported (0x80004002 (E_NOINTERFACE)).
   at Microsoft.Web.WebView2.Core.CoreWebView2WebMessageReceivedEventArgs.get__nativeICoreWebView2WebMessageReceivedEventArgs2()

**Comment by d2phap** [comment]
Hi @SkullRunnerGR 

Can you share
- Steps to reproduce the error?
- What folder did you extract?
- What is version of MS Edge?
